### ISSUE TITLE
OpsView Connections Errors

### ISSUE THREAD

**Issue by MarcusCaepio** [HenriWahl/Nagstamon#207]
When I connect to a external OpsView Server, Nagsta tells me a Connection Error in a few seconds intervall. The message appears, disappears, appears, disappears and so on.

**Comment by HenriWahl** [comment]
Strange, I do have the same with only one Check_MK machine. Let's see... how is your update interval? What network connection is this - LAN or WAN?

**Comment by MarcusCaepio** [comment]
It's WAN. There is no problem with LAN Server

**Comment by HenriWahl** [comment]
Any proxy involved?

**Comment by MarcusCaepio** [comment]
Nope

**Comment by HenriWahl** [comment]
I get such output in the console:

Traceback (most recent call last):
  File "/Nagstamon/Servers/Generic.py", line 1196, in FetchURL
    response = self.session.get(url, timeout=30)
  File "/usr/lib/python3.4/site-packages/requests/sessions.py", line 480, in get
    return self.request('GET', url, **kwargs)
  File "/usr/lib/python3.4/site-packages/requests/sessions.py", line 468, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python3.4/site-packages/requests/sessions.py", line 576, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python3.4/site-packages/requests/adapters.py", line 426, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', BadStatusLine("''",))

The same with you?

**Comment by MarcusCaepio** [comment]
How do I check the console? ^^

**Comment by HenriWahl** [comment]
Run it from console... how do you start Nagstamon? Even possible in Windows to run in console but then you would need to install some devel stuff described at https://nagstamon.ifw-dresden.de/nagstamon-20/. If this is not possible enable debugging on first tab of settings dialog and lets hope there will be anything in an outputfile.

**Comment by MarcusCaepio** [comment]
I start it via .exe which is provided by the build.py. Created the build of the current git this morning. Running Nagstamon on Win 7 x64

**Comment by HenriWahl** [comment]
Ah so you already have the build environment! Great. So just go into the root directory of you Nagstamon sources and run nagstamon.py in the console.

**Comment by MarcusCaepio** [comment]
There is no nagstamon.py only nagstamon.exe

**Comment by HenriWahl** [comment]
The same as I get... I googled around to find this being a bug in httplib which Requests uses

**Comment by MarcusCaepio** [comment]
And you also know the bugfix? ^^

**Comment by HenriWahl** [comment]
Nope :-(

**Comment by HenriWahl** [comment]
One workaround might be to ignore exactly this error - but I really do not understand what exactly it means.

**Comment by HenriWahl** [comment]
I looked deeper at this but what I found does not give much hope:

https://github.com/kennethreitz/requests/issues/2510
https://github.com/kennethreitz/requests/issues/1915

They all point to the server sending invalid data. Which is strange because in my case some connection attempts work, Nagstamon gets and displays information, then some connection errors appear and vanish again. Still no clue what the best strategy will be.
